validate.py

```diff
@@ -86,6 +87,49 @@ def get_reviews():
 def get_users():
   return list(sorted(os.listdir('players/')))
 
+def get_user_points():
+  points = {}
+
+  for user in get_users():
+    points[user] = 0
+
+    bonus_directory = os.path.join('players', user, 'bonuses')
+    if os.path.isdir(bonus_directory):
+      for named_bonus in os.listdir(bonus_directory):
+        with open(os.path.join(bonus_directory, named_bonus)) as inf:
```

Comment:
This throws an error if `named_bonus` is a directory; a player could win just by creating a directory called "<user> is a winner".